docs/frequently-asked-questions.md

```diff
@@ -4,15 +4,19 @@
 
 - [Frequently Asked Questions](#frequently-asked-questions)
   - [Links/Graphs/BackLinks don't work. How do I enable them?](#linksgraphsbacklinks-dont-work-how-do-i-enable-them)
+  - [I don't want Foam enabled for all my workspaces](#i-dont-want-foam-enabled-for-all-my-workspaces)
 
 ## Links/Graphs/BackLinks don't work. How do I enable them?
 
 - Ensure that you have all the [[recommended-extensions]] installed in Visual Studio Code
 - Reload Visual Studio Code by running `Cmd` + `Shift` + `P` (`Ctrl` + `Shift` + `P` for Windows), type "reload" and run the **Developer: Reload Window** command to for the updated extensions take effect
 - Check the formatting rules for links on [[foam-file-format]], [[wiki-links]] and [[link-formatting-and-autocompletion]]
 
+## I don't want Foam enabled for all my workspaces
+Any extension you install in Visual Studio Code is enabled by default. Give the philosophy of Foam it works out of the box without doing any configuration upfront. In case you want to disable Foam for a specific workspace, or disable Foam by default and enable it for specific workspaces, it is advised to follow the best practices as [documented by Visual Studio Code](https://code.visualstudio.com/docs/editor/extension-marketplace#_manage-extensions)
+
 [//begin]: # "Autogenerated link references for markdown compatibility"
-[recommended-extensions]: recommended-extensions.md "Recommended Extensions"
-[foam-file-format]: dev/foam-file-format.md "Foam File Format"
-[wiki-links]: wiki-links.md "Wiki Links"
+[recommended-extensions]: recommended-extensions "Recommended Extensions"
```

Comment:
This should be with extension

Reply:
Woops, it went automatically so I'll check the config file in the repo as well. Changed!